Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py

```diff
@@ -157,8 +157,27 @@ def process_item(cls, threat: dict) -> List[dict]:
             block["value"] = value
             block["type"] = indicator
             block["threat_id"] = threat_id
+            malware_family: dict = block.get("malwarefamily", {})
+            ip_detail: dict = block.get("ipDetail", {})
             if indicator:
-                results.append({"value": value, "type": indicator, "rawJSON": block})
+                results.append({
+                    "value": value,
+                    "type": indicator,
+                    "rawJSON": block,
+                    "fields": {
+                        "name": threat_id,
+                        "malwarefamily": malware_family.get("familyName"),
+                        "description": malware_family.get("description"),
+                        "feedoriginalseverity": block.get("impact"),
```

Comment:
Please check what happens if these values are `None`. If it's problematic then you'll need to set the default argument to `get()` to be of the appropriate type that given indicator field expects.

Reply:
Checked, no exception is raised. Indicators with missing fields are being fetched as usual.